src/monero_send_routine.cpp

```diff
@@ -426,6 +426,7 @@ void _reenterable_construct_and_send_tx(
 			args.fee_per_b,
 			args.fee_quantization_mask,
 			tie_outs_to_mix_outs_retVals.mix_outs,
+			200,
```

Comment:
1. Can this be a named constant?
2. Is there a reason we are adding a new parameter in the middle of the argument list instead of at the end?

Reply:
Hi @feri42 ,
> Can this be a named constant?
Sure, I improved by replacing with a constant

> Is there a reason we are adding a new parameter in the middle of the argument list instead of at the end?
I just reverted this old logic to support subaddress at [PR#20](https://github.com/ExodusMovement/mymonero-core-cpp/pull/20)